build.sbt

```diff
@@ -114,7 +114,7 @@ libraryDependencies ++= Seq(
   "org.scalamock"               %% "scalamock"                  % "4.2.0" ,
   "org.scalatestplus"           %% "scalatestplus-scalacheck"   % "3.1.0.0-RC2",
   "com.vladsch.flexmark"         % "flexmark-all"               % "0.62.2" ,
-  "uk.gov.hmrc"                 %% "reactivemongo-test"         % s"5.0.0-$playVersion",
```

Comment:
reactive mongo was not being used in the test cases so instead of upgrading it , you can remove it